Shared/Samples/Show utility associations/ShowUtilityAssociationsView.swift

```diff
@@ -0,0 +1,271 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowUtilityAssociationsView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The current viewpoint of the map.
+    @State var viewpoint: Viewpoint = .initialViewpoint
+    
+    /// The scale of the viewpoint.
+    @State var scale: Double = .zero
+    
+    /// An image that represents an attachment symbol.
+    @State private var attachmentImage: UIImage?
+    
+    /// An image that represents a connectivity symbol.
+    @State private var connectivityImage: UIImage?
+    
+    /// The display scale of this environment.
+    @Environment(\.displayScale) private var displayScale
+    
+    var body: some View {
+        MapView(
+            map: model.map,
+            graphicsOverlays: [model.associationsOverlay]
+        )
+        .onScaleChanged {
+            scale = $0
+            Task { try await model.addAssociationGraphics(viewpoint: viewpoint, scale: scale) }
+        }
+        .onViewpointChanged(kind: .boundingGeometry) {
+            viewpoint = $0
+            Task { try await model.addAssociationGraphics(viewpoint: viewpoint, scale: scale) }
+        }
+        .task {
+            await model.setup()
+            do {
+                try await model.addAssociationGraphics(viewpoint: viewpoint, scale: scale)
+            } catch {
+                return
+            }
```

Comment:
Now that `isAuthenticated` is a computed property, there is a timing issue where the graphics are added initially before the sample is authenticated, so there are no graphics in the sample until the scale or viewpoint changes and the graphics are updated when called in the model. Adding the graphics in the task block here works. Is there a better way to address this timing issue?